account_asset_management/models/account_asset.py

```diff
@@ -532,8 +552,11 @@ def validate(self):
                 asset.state = "close"
             else:
                 asset.state = "open"
-                if not asset.depreciation_line_ids.filtered(
-                    lambda l: l.type != "create"
+                if (
+                    not asset.depreciation_line_ids.filtered(
+                        lambda l: l.type != "create"
+                    )
+                    and asset.method != "unit-activity"
```

Comment:
I think it maybe better to override `compute_depreciation_board` and by pass the whole method for method = unit-activity, because it could be called at other places, what do you think ?